frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx

```diff
@@ -80,10 +42,15 @@ const SelectCapacityAndNodes: React.FC<SelectCapacityAndNodesProps> = ({
 }) => {
   const { t } = useTranslation();
 
+  React.useEffect(() => {
+    dispatch({ type: 'capacityAndNodes/capacity', payload: '2Ti' });
```

Comment:
If I select let's say "0.5" value in this step, then moves to step-3. When I will come "Back" to this step again "2" will be pre-selected. May be we do not what to fix it ? Still I think this issue will happen here.

Reply:
right, updated!